### ISSUE TITLE
2.0.0 – 'ABKAppboyEndpointDelegate.h' file not found

### ISSUE THREAD

**Issue by ldiqual** [Appboy/appboy-segment-ios#13]
```
pod 'Appboy-iOS-SDK', '3.0.1'
pod 'Segment-Appboy', '2.0.0'
```

```
/path/to/app/Pods/Segment-Appboy/Pod/Classes/SEGAppboyIntegrationEndpointDelegate.h:2:9: 'ABKAppboyEndpointDelegate.h' file not found
```

**Comment by chshapiro** [comment]
Hey @ldiqual,

Thank you for reporting this issue! We have tried reproducing but cannot seem to get the same error message. Can you provide us with some more information about how you integrate and use Segment-Appboy: 
- What version of Cocoapods are you using? 
- Are you using `use_frameworks!` in your Podfile? 
- Did you set a custom endpoint on Segment's dashboard? 
- At what point are you getting this error?

Thank you,
Chava

**Comment by JamesJSY** [comment]
@chshapiro 
Hi Chava!

I am having the same issue. For the questions -
1. 1.3.1
2. Yes
3. Nope
4. pod install is all good. Error occurs when building the project

One difference is that i use 3.0.2 as the Appboy-iOS-SDK

Looking forward to hearing from you!

Thank you!
Shiyuan

**Comment by chshapiro** [comment]
Hey @JamesJSY,

We just released version 2.0.1 which we think should fix this issue. Can you test it out and let us know if you still have any problems?

Thanks,
Chava

**Comment by JamesJSY** [comment]
Thanks @chshapiro! Works now :D
